Tests/SonarScanner.MSBuild.PreProcessor.Test/SonarWebServiceTest.cs

```diff
@@ -682,60 +682,6 @@ public void TryDownloadEmbeddedFile_RequestedFileDoesNotExist()
             File.Exists(expectedFilePath).Should().BeFalse("File should not be created");
         }
 
-        [TestMethod]
-        public async Task DownloadCache_NullArguments()
```

Comment:
Info: The tests were moved to SonarQubeWebService and SonarCloudWebService since the implementation is product specific.